client/components/plans/plan/index.jsx

```diff
@@ -249,6 +250,21 @@ module.exports = React.createClass( {
 		);
 	},
 
+	getFreeDomainMessage: function() {
+		if ( abtest( 'promoteFreeDomain' ) !== 'freeDomain' ) {
+			return null;
+		}
+
+		if ( this.props.plan && isPlan( this.props.plan ) ) {
+			return (
+				<div className="plan__free-domain-message">
+					<Gridicon icon="checkmark" size="12" />
```

Comment:
I think you only need curly braces when the thing you are passing is a variable, or other encapsulated JavaScript.